app/views/users/_vitals.html.erb

```diff
@@ -0,0 +1,3 @@
+        <i class="glyphicon glyphicon-time"></i> Joined on <%= @user.created_at.to_date.strftime("%b %d, %Y") %> <br />
+        <i class="glyphicon glyphicon-folder-open"></i> Deposited Files
+        <%= link_to number_of_deposits(@user), search_action_path(render_facet_params("depositor_ssim", @user.to_s)) %> <br />
```

Comment:
Is there a more straight forward fix for this? What do you think of something like

```
diff --git a/app/helpers/sufia_helper.rb b/app/helpers/sufia_helper.rb
index b02c5d7..f192d07 100644
--- a/app/helpers/sufia_helper.rb
+++ b/app/helpers/sufia_helper.rb
@@ -2,4 +2,11 @@ module SufiaHelper
   include ::BlacklightHelper
   include Sufia::BlacklightOverride
   include Sufia::SufiaHelperBehavior
+
+  #overload link_to_facet to display the label we want
+  def link_to_facet(field, field_string, label = nil)
+    label ||= field
+    path = add_facet_params(field_string, field).merge!(controller: "catalog", action: "index")
+    link_to(label, path)
+  end
 end
diff --git a/app/views/users/_vitals.html.erb b/app/views/users/_vitals.html.erb
index 3ba557a..fe795dc 100644
--- a/app/views/users/_vitals.html.erb
+++ b/app/views/users/_vitals.html.erb
@@ -1,3 +1,3 @@
         <i class="glyphicon glyphicon-time"></i> Joined on <%= @user.created_at.to_date.strftime("%b %d, %Y") %> <br />
         <i class="glyphicon glyphicon-folder-open"></i> Deposited Files
-        <%= link_to number_of_deposits(@user), search_action_path(render_facet_params("depositor_ssim", @user.to_s)) %> <br />
+        <%= link_to_facet(@user.to_s, Solrizer.solr_name("depositor", :symbol), number_of_deposits(@user)) %> <br />
```